pcsx2/SIO/Memcard/MemoryCardFile.cpp

```diff
@@ -987,7 +981,8 @@ bool FileMcd_CreateNewCard(const std::string_view& name, MemoryCardType type, Me
 			{
 				if (std::fwrite(buf, sizeof(buf), 1, fp.get()) != 1)
 				{
-					Host::ReportFormattedErrorAsync("Memory Card Creation Failed", "Failed to write file '%s'.", full_path.c_str());
+					Host::ReportErrorAsync(TRANSLATE_SV("MemoryCard", "Memory Card Creation Failed"),
```

Comment:
```suggestion
					Host::ReportErrorAsync("Memory Card Creation Failed",
```
don't mix translated and non-translated messages